src/main/java/seedu/address/model/assignment/DueDate.java

```diff
@@ -2,28 +2,59 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
 
 public class DueDate {
     public static final String MESSAGE_CONSTRAINTS = "Due dates should be in a format dd/MM/yyyy HHmm";
-    //TODO: decide which parse, output format to keep
-    protected static final DateTimeFormatter PARSE_FORMAT = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+
+    protected static final DateTimeFormatter PARSE_DATE_FORMAT = DateTimeFormatter.ofPattern("d/M/yyyy");
```

Comment:
why the pattern here is "d/M/yyyy" instead of "dd/MM/yyyy"

Reply:
I can address this part since I wrote out this format.

"d" means you can write the date as "06" or "6". 
If we use "dd", then all single digit dates must have a trailing zero like "06". If not, we will get a ```DateTimeParseException```

Same goes for the month formatting.